pkg/assets/apps.go

```diff
@@ -55,6 +57,10 @@ func (d *dpApplier) Reader(objBytes []byte, render RenderFunc, params RenderPara
 		panic(err)
 	}
 	d.dp = obj.(*appsv1.Deployment)
+	if d.dp.Spec.Template.ObjectMeta.Annotations == nil {
+		d.dp.Spec.Template.ObjectMeta.Annotations = make(map[string]string)
+	}
+	d.dp.Spec.Template.ObjectMeta.Annotations["operator.openshift.io/force"] = uuid.NewString()
```

Comment:
I wonder if this is an annotation string we should use. Function in library-go is deprecated so I wouldn't consider this to an established convention and without `microshift` in there it could send someone on a trip in wrong direction.

Reply:
Good point about the name.